code/modules/clothing/clothing.dm

```diff
@@ -88,23 +88,27 @@
 	else
 		return ..()
 
-/obj/item/clothing/attackby(obj/item/W, mob/user, params)
-	if(W.get_sharpness() && cuttable)
-		if (alert(user, "Are you sure you want to cut the [src] into strips?", "Cut clothing:", "Yes", "No") != "Yes")
+/obj/item/clothing/attackby(obj/item/tool, mob/user, params)
+	if(tool.get_sharpness() && cuttable)
+		if(tgui_alert(user, "Are you sure you want to cut the [src] into strips?", "Cut clothing:", list("Yes", "No")) != "Yes")
```

Comment:
the [src] will lead to "the the" since [src] includes the in some cases

Reply:
```suggestion
		if(tgui_alert(user, "Are you sure you want to cut \the [src] into strips?", "Cut clothing:", list("Yes", "No")) != "Yes")
```